src/html/fetch-content.js

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2018 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+const { setdefault } = require('ferrum');
+
+async function fetchContent(context, {
+  request, downloader, secrets, logger,
+}) {
+  setdefault(context, 'content', {});
+  /* istanbul ignore next */
+  if (!request || !request.params) {
+    throw new Error('Request parameters missing');
+  }
+  const { content } = context;
+  const {
+    owner, repo, path, branch = 'master', ref,
+    contentProxyUrl = 'https://adobeioruntime.net/api/v1/web/helix/helix-services/content-proxy@v1',
```

Comment:
@trieloff this would be set by VCL to use the correct runtime namespace. we want to run the content-proxy in the customers namespace.

Reply:
I'd take the namespace from `__ow` so that we don't need VCL changes.

Reply:
hmmm....good point :-)